charts/pega/charts/hazelcast/templates/pega-hz-deployment.yaml

```diff
@@ -25,6 +25,9 @@ spec:
   {{- if ( .Values.imagePullPolicy ) }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
   {{- end }}
+        volumeMounts:
+        - name: {{ template "pegaVolumeCredentials" }}
+          mountPath: "/opt/hazelcast/secrets"
```

Comment:
Do we need to create this folder `/opt/hazelcast/secrets` inside the docker container in pega-web-ready image or it just works by default? Please check this reference.
https://github.com/pegasystems/docker-pega-web-ready/blob/master/Dockerfile#L58

Reply:
opt/hazelcast/secrets folder is being created from docker-entrypoint.sh of clustering-service
https://git.pega.io/projects/PS/repos/clustering-service/pull-requests/9/diff#distribution/docker/src/main/resources/docker-entrypoint.sh?t=8

Reply:
At the time the entrypoint runs, the secret will either be injected already (or not).

Reply:
I think volume mount happens before entrypoint. I doubt if creating folder in entrypoint has any affect. If user has required permissions then folders get automatically created during mount.

Reply:
Will verify this by reverting the changes made in clustering-service repo